src/shared/components/navbar/desktop/index.module.css

```diff
@@ -31,11 +32,12 @@
             transition: var(--transition-default-duration) ease;
 
             &:nth-child(4) {
-                margin-right: 10px;
+                margin-right: 15px;
             }
 
             &:hover {
                 color: var(--color-yellow);
+                cursor: pointer;
```

Comment:
If you use `<a>` like I said, we don't need this. If not, please move the `cursor: pointer` to non-hover state.